py_zipkin/encoding/_encoders.py

```diff
@@ -298,3 +301,32 @@ def encode_span(self, span):
         encoded_span = json.dumps(json_span)
 
         return encoded_span
+
+
+class _V2ProtobufEncoder(IEncoder):
+    """Protobuf encoder for V2 spans."""
+
+    def fits(self, current_count, current_size, max_size, new_span):
+        """Checks if the new span fits in the max payload size.
+
+        We can't encode protobuf spans one by one and just sum their sizes as we
```

Comment:
I think this is actually not true. see https://github.com/openzipkin/zipkin/blob/master/zipkin/src/main/java/zipkin2/codec/Encoding.java#L63

Reply:
Before I was trying to convert each span to bytes and concatenate them which didn't work. Then I realized that if I encoded them as 1 element ListOfSpans I could very easily concatenate everything